Specifications/Projections/Builder/Copies/MongoDB/for_ConversionsFromBsonClassMapAdder/when_adding_from_class_map/from_lookup_for_read_model_type/with_pascal_case_id_field.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Machine.Specifications;
+using MongoDB.Bson.Serialization;
+using It = Machine.Specifications.It;
+
+namespace Dolittle.SDK.Projections.Builder.Copies.MongoDB.for_ConversionsFromBsonClassMapAdder.when_adding_from_class_map.from_lookup_for_read_model_type;
+
+public class with_pascal_case_id_field : given.all_dependencies
+{
+    public class read_model
+    {
+        public string Id;
+        public string Field;
+        public string Property { get; }
+    }
+
+    Because of = () => adder.AddFrom(BsonClassMap.LookupClassMap(typeof(read_model)), build_results, conversions.Object);
```

Comment:
It would be nice if these specs passed in a mocked ClassMap, but we can change it later.

Reply:
I was thinking of having that as additional specs to these ones with the types maybe?